pyproject.toml

```diff
@@ -0,0 +1,4 @@
+[tool.black]
+line-length = 120
+target-version = ["py38", "py39"]
```

Comment:
Wouldn't it be better if we add py310?

Reply:
Related to this, I saw the code was styled to python 3.8, isn't it better if we do it with the latest version?

Reply:
We can bump that when we switch to a newer version, the main repo is using python 3.8. If we set a newer syntax black can format the code in ways older versions don't understand (not very likely but can happen)